docs/zh-CN/sql-reference/sql-statements/Administration/CLEAN TRASH.md

```diff
@@ -0,0 +1,47 @@
+---
+{
+    "title": "CLEAN TRASH",
+    "language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# CLEAN TRASH
+## description
+    该语句用于清理 backend 内的垃圾数据。
```

Comment:
Will this statement clean up both trash and snapshot?
Will cleaning up the snapshot involve the snapshot being restored?

Reply:
> 
> 
> Will this statement clean up both trash and snapshot?
> Will cleaning up the snapshot involve the snapshot being restored?

Yes, this statement will clean up both trash and snapshot.

This statement will call `StorageEngine::start_trash_sweep`.
It will only clean up expired data (define at `config::snapshot_expire_time_sec`/`config::trash_file_expire_time_sec`).
And this function will be automatically called periodically, so I think its cleanup is harmless.